rich/_console.py

```diff
@@ -0,0 +1,141 @@
+CONSOLE_HTML_FORMAT = """\
```

Comment:
As these 2 `CONSOLE_` variables take quite a lot of space in the `console` module (especially the SVG one) I extracted them to this private module to make the work on the `console` module easier (less scrolling to navigate in this rather long file is always good to have right? :smile: )

If anyone was using them in their own code, with `from rich.console import CONSOLE_SVG_FORMAT` for example, this will still work as these 2 consts are imported in `console.py`